tools/runme/build.sh

```diff
@@ -326,10 +286,9 @@ _full_build() {
   # tests
   should test python     && @ "../pytests/auto-tests"
   should test python     && @ "../pytests/notebook-tests"
-  should test e2e        && _e2e_tests
+  should test e2e        && _sbt_adb_notebooks
```

Comment:
Re the name: using `e2e` should mean that this line doesn't change.

Reply:
there will be other e2e tests, so will keep this separate

Reply:
(Same as above)